app/models/user.rb

```diff
@@ -1,5 +1,7 @@
 class User < ApplicationRecord
   has_many :trips, dependent: :destroy
+  has_many :reviews, through: :trips
```

Comment:
Is there any situation where we would need to see all of a user's reviews?

Reply:
Would a user want to view all his reviews?